### ISSUE TITLE
recurate PMID:37164017

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3716]
It has been on my list to recurate this.
Snezka curated for japonicusDB, and I didn't check it  but I wonder if we can use some of the metabolomics to convert inferred TCA and glycolysis enzymes to published with EXP

https://www.pombase.org/reference/PMID:37164017

**Comment by ValWood** [comment]
Snezka also provided a note about not being able to curate some things from this paper. I  was positive a  "message to curators" but I can't see one in either the PomBase or Japonicus Canto?

**Comment by ValWood** [comment]
@kimrutherford  has PMID:37164017 ever had a message to curators?

**Comment by ValWood** [comment]
I found the message in e-mail:
I did what I could for both PomBase and JaponicusDB - but I realized that we miss a lot of metabolic terms, for instance related to the TCA cycle. I tried TCA, Krebs, tricarboxylic acid cycle, but could not find the terms. Or even things like "increased/decreased biomass yield coefficient", "increased glucose consumption", altered ATP levels, or "increased / decreased metabolite synthesis rate" - note that the latter is not interchangeable with metabolite abundance/levels.
